spirv_glsl.hpp

```diff
@@ -399,6 +399,8 @@ class CompilerGLSL : public Compiler
 	void end_scope_decl(const std::string &decl);
 
 	Options options;
+	bool declaredKHR_shader_subgroup_basic = false;
```

Comment:
Should be `declared_KHR_shader_subgroup_basic`, not camel case. Since there is an entry point for emitting extension workarounds, probably best to hoist all these members into an anonymous struct.

Reply:
@HansKristian-Work I dont like that approach anyway, because we should analyze the ops used and not the extensions declared because they dont map 1:1 with desktop GL